openwisp_controller/config/utils.py

```diff
@@ -199,3 +201,49 @@ def get_default_templates_queryset(
     if backend:
         queryset = queryset.filter(backend=backend)
     return queryset
+
+
+def get_api_urls(api_views):
```

Comment:
why is this function being added here? I think it should go in `openwisp_controller.config.api.urls`.

BTW the reasons we use functions is to allow to extende the API views and URLs but I don't see anything done on that front but I think we should.

Reply:
Okay, will move it to `openwisp_controller.config.api.urls`